lib/index.ts

```diff
@@ -274,7 +277,31 @@ export async function exec(argv2: string[]) {
 
   const forceBuild = argv.b || argv.build;
 
-  // _
```

Comment:
The section comment `// _` is probably replaced erroneously. this should be moved to line 305 instead.